includes/discovery/mempools/moxa-etherdevice.inc.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/*
+ * LibreNMS Moxa EtherDevice RAM discovery module
+ *
+ * Copyright (c) 2017 Aldemir Akpinar <aldemir.akpinar@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+
+
+if ($device['os'] == 'moxa-etherdevice') {
+    echo 'Moxa EtherDevice';
+
+    $total = str_replace('"', "", snmp_get($device, "MOXA-IKS6726A-MIB::totalMemory.0", '-OvQ'));
```

Comment:
If you move the mib into mibs/moxa then you can replace the snmp queries with:

`snmp_get($device, "totalMemory.0", '-OvQ', 'MOXA-IKS6726A-MIB')`